src/cmd/services/m3query/config/config.go

```diff
@@ -270,6 +270,9 @@ type QueryConfiguration struct {
 	// RestrictTags is an optional configuration that can be set to restrict
 	// all queries with certain tags by.
 	RestrictTags *RestrictTagsConfiguration `yaml:"restrictTags"`
+	// RequireStartTime requires requests to /label(s) and /series endpoints
+	// to specify a start time to prevent unbounded queries.
+	RequireStartTime bool `yaml:"requireStartTime"`
```

Comment:
Maybe we separate these out to two config values?

That way it can be more easily separated and we don’t have to deprecate this field.

e.g. requireLabelsEndpointStartEndTime and requireSeriesStartEndTime